vmdb/tools/fix_auth/fix_auth.rb

```diff
@@ -73,7 +73,7 @@ def fix_database_passwords
 
     def fix_database_yml
       FixDatabaseYml.file_name = "#{options[:root]}/config/database.yml"
-      FixDatabaseYml.run(run_options.merge(:hardcode => options[:password]))
+      FixDatabaseYml.run({:hardcode => options[:password]}.merge(run_options))
```

Comment:
@abellotti no.
They can specify `--hardcode` OR `--password`
There was a bug before that would ignore `--hardcode PASSWORD` if they specified it.

I guess I could have written:
```ruby
def fix_database_yml
  FixDatabaseYml.file_name = "#{options[:root]}/config/database.yml"
  FixDatabaseYml.run(run_options(:hardcode => (options[:password] || options[:hardcode])`)
end
```
but since `:hardcode` is already in there, I tack it on if it isn't already set.
Alternately I guess could have used `reverse_merge()`